src/compiler/watchUtilities.ts

```diff
@@ -257,6 +257,30 @@ namespace ts {
         Full
     }
 
+    /**
+     * Updates the map of extended config file watches with a new set of extended config files from a base config file
+     */
+    export function updateExtendedConfigFilesMap(
+        configFile: TsConfigSourceFile,
+        extendedConfigFilesMap: ESMap<string, FileWatcher>,

```

Comment:
This needs to be `ESMap<string, FileWatcher>`

Reply:
It's already `ESMap<string, FileWatcher>`. After reading [this other comment](https://github.com/microsoft/TypeScript/pull/41493#discussion_r533856096) do you want to change it to `ESMap<Path, FileWatcher>` and use the `Path` of the extended configs as the key?

Reply:
Sorry that i didnt realise that i typed the same type. Yes we want them to be Path as key since those are the keys for any kind of file paths